docs/_files/id-repository-config-files/README.md

```diff
@@ -0,0 +1,12 @@
+# ID REPOSITORY SERVICES CONFIGURATION FILES
+
+## Overview
+
+This folder contains the files required for local configurations of Resident services.
```

Comment:
This folder contains the files required for local configurations of Resident services.    --- why Resident services?